concepticondata/conceptlists.tsv

```diff
@@ -239,3 +239,4 @@ Carling-2017-97j	Carling, Gerd and Larsson, Filip and Cronhamn, Sandra and Farre
 Carling-2017-87k	Carling, Gerd and Larsson, Filip and Cronhamn, Sandra and Farren, Rob and Aliyev, Elnur and Johansson, Niklas and Chukhua, Merab and Goergens, Anne and Vamling, Karina and Cathcart, Chundra and Tetraze, Maka and Lomadze, Tamuna and Holmer, Arthur and Avidzba, Leila and Gvantseladze, Teimuraz and Khalilov, Madzhid and Cabral, Ana Suelly Arruda Câmara and Mana, Joaquim and Sinha, Vera da Silva and Sampaio, Wany and Sinha, Chris and Gakran, Nanblá and Sabino, Wary Kamaiurá and Kamaiura, Aisanain Páltu and Nahukua, Kaman and Mehinako, Makaulaka	2017	k	87	basic,areal	English	Turkic languages	https://diacl.ht.lu.se/WordList/Index	Carling2017		A list of basic cultural terms for the Turkic languages.		
 Nagano-2013-1256	Nagano, Yasuhiko and Prins, Marielle	2013		1256	questionnaire,areal	English	South-East Asian Languages	https://htq.minpaku.ac.jp/databases/rGyalrong/lang/area/index.php?langindex=eng	Nagano2013		The results of an international joint survey of the rGyalrongic languages, originally published by the authors in their own website.		
 Yang-2011-1014	Yang, Cathryn	2011		1014	questionnaire	Chinese	Lalo Languages	https://www.sil.org/system/files/reapdata/45/63/47/45634767234331504727329755903316120678/Yang_Lalo_Sept_2011.pdf	Yang2011		A survey of Lalo regional varieties for research in phylogenetics, dialectometry and sociolinguistics.		
+Vejdemo-2016-167	Vejdemo, Susanne and Hörberg, Thomas	2016		167	basic	English	Global	https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0147924#pone.0147924.ref032 	Vejdemo2016		This list represents 167 content words, derived from [the 200-item Swadesh list (1952)](:ref:Swadesh-1952-200), with a few concepts coming from  [Swadesh’s list of 215 items (1955)](:ref:Swadesh-1955-215). It was originally compiled with the purpose of assessing the rate of lexical replacement for the specified concepts in English, Danish, Dutch, German and Swedish, according to word class, frequency, age of acquisition, synonyms, arousal, imageability and average mutual information either from published databases or from corpora and lexica.		
```

Comment:
`Swadesh’s` should be spelled with `'` instead of `’` as we have been using normal apostroph throughout.